ee/server/service/teams.go

```diff
@@ -1638,44 +1648,53 @@ func unmarshalWithGlobalDefaults(b *json.RawMessage) (fleet.Features, error) {
 	return *defaults, nil
 }
 
-func (svc *Service) updateTeamMDMDiskEncryption(ctx context.Context, tm *fleet.Team, enable *bool) error {
-	var didUpdate bool
+func (svc *Service) updateTeamMDMDiskEncryption(ctx context.Context, tm *fleet.Team, enable *bool, requirePIN *bool) error {
+	var didUpdateEncryption bool
+	var didUpdateRequirePIN bool
 	if enable != nil {
 		if tm.Config.MDM.EnableDiskEncryption != *enable {
 			if *enable && svc.config.Server.PrivateKey == "" {
 				return ctxerr.New(ctx, "Missing required private key. Learn how to configure the private key here: https://fleetdm.com/learn-more-about/fleet-server-private-key")
 			}
 
 			tm.Config.MDM.EnableDiskEncryption = *enable
-			didUpdate = true
+			didUpdateEncryption = true
+		}
+	}
+	if requirePIN != nil {
+		if tm.Config.MDM.RequireBitLockerPIN != *requirePIN {
+			tm.Config.MDM.RequireBitLockerPIN = *requirePIN
 		}
+		didUpdateRequirePIN = true
```

Comment:
Shouldn't this be `true` iff `tm.Config.MDM.RequireBitLockerPIN != *requirePIN` ?